### ISSUE TITLE
Move Instagram & Twitter api credentials to DB

### ISSUE THREAD

**Issue by anirudh-eka** [anirudh-eka/dashtag#112]
We need to be able to encrypt and decrypt them

**Comment by pturley** [comment]
I dont really know how much we are gaining if we have an app that knows how to encrypt and decrypt this content.  Normally you only build an app that knows how to encrypt things and then it can tell you if two things are the same, but it cant tell you what the thing in the DB is.

I honestly think we should keep this stuff in cleartext for now.

I think that the only security profile you would be guarding against is if someone gained access to your database without gaining access to your application, and I think there are better ways to deal with that security concern to be honest.  

What do you think about what I am saying?

**Comment by anirudh-eka** [comment]
> I think that the only security profile you would be guarding against is if someone gained access to your database without gaining access to your application

Yes, you're right. What are the better ways to deal with that concern?

**Comment by pturley** [comment]
Make it such that there is no way to get to the database other than through the application by shutting off all access to any other port from any other address.
